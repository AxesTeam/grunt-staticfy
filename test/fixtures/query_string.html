<!DOCTYPE html>
<html>
<head lang="en">
    <meta charset="UTF-8">
    <title></title>
</head>
<body>
<script type="text/javascript">
    document.writeln(location.search);
</script>
</body>
</html>