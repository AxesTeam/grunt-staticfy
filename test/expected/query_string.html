<!DOCTYPE html><html><head lang="en">
    <meta charset="UTF-8">
    <title></title>
</head>
<body>
<script type="text/javascript">
    document.writeln(location.search);
</script>?code=1


</body></html>